src/components/resumes/ResumeList/ResumeList.jsx

```diff
@@ -0,0 +1,55 @@
+import Box from "./Box";
+import Button from "@material-ui/core/Button";
+import GetResume from "./ResumeListAPI.js";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableContainer from "@material-ui/core/TableContainer";
+import { makeStyles } from "@material-ui/core/styles";
+import React, { useEffect, useState } from "react";
+import "./ResumeList.css";
+
+const useStyles = makeStyles({
+  table: {
+    maxWidth: 900,
+    minWidth: 150,
+  },
+});
+
+function ResumeList() {
+  const classes = useStyles();
+  const [resumes, setResumes] = useState([]);
+
+  const refresh = () => {
```

Comment:
You're not reusing this function why you created this instead of placing the body of this function directly on the useEffect body?